e2e-tests/submit-to-trade-finance-manager/cypress/integration/journeys/portal/MIA-underwriter-managers-decision-updates-portal-deal/TFM-MIA-underwriter-managers-decision-approved-with-conditions-updates-portal-deal.spec.js

```diff
@@ -107,13 +105,5 @@ context('Portal to TFM deal submission', () => {
     // Portal deal comments/conditions should be displayed
     //---------------------------------------------------------------
     portalPages.contract.commentsTab().click();
-
-    portalPartials.ukefComments.specialCondition.title().invoke('text').then((text) => {
-      expect(text.trim()).to.equal('Special Conditions:');
-    });
-
-    portalPartials.ukefComments.specialCondition.text().invoke('text').then((text) => {
-      expect(text.trim()).to.equal(MOCK_COMMENTS);
-    });
```

Comment:
I think this might have been incorrect - should be testing regular comments rather than special condition - can help with this PR if you want :)